specification/appcomplianceautomation/resource-manager/Microsoft.AppComplianceAutomation/preview/2022-11-16-preview/appcomplianceautomation.json

```diff
@@ -0,0 +1,1374 @@
+          }
+        },
+        "nextLink": {
+          "description": "The URL the client should use to fetch the next page (per server side paging).\r\nIt's null for now, added for future use.",
+          "type": "string"
+        }
+      }
+    },
+    "ResourceItem": {
+      "type": "object",
+      "description": "Resource Id.",
+      "properties": {
+        "subscriptionId": {
+          "description": "The subscription Id of this resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceGroup": {
+          "description": "The resource group name of this resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceType": {
+          "description": "The resource type of this resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "resourceId": {
+          "description": "The resource Id.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DownloadResponse": {
+      "description": "Object that includes all the possible response for the download operation.",
+      "type": "object",
+      "properties": {
+        "resourceList": {
+          "description": "List of the reports",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ResourceItem"
+          },
+          "x-ms-identifiers": []
+        },
+        "complianceReport": {
+          "description": "List of the compliance result",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ComplianceReportItem"
+          },
+          "x-ms-identifiers": []
+        },
+        "compliancePdfReport": {
+          "description": "compliance pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance pdf report",
+              "type": "string",
+              "readOnly": true,
+              "x-ms-secret": true
+            }
+          }
+        },
+        "complianceDetailedPdfReport": {
+          "description": "compliance detailed pdf report",
+          "type": "object",
+          "readOnly": true,
+          "properties": {
+            "sasUri": {
+              "description": "uri of compliance detailed pdf report",
+              "type": "string",
+              "readOnly": true
+            }
+          }
+        }
+      }
+    },
+    "ComplianceReportItem": {
+      "type": "object",
+      "description": "Object that includes all the content for single compliance result.",
+      "properties": {
+        "categoryName": {
+          "type": "string",
+          "description": "The category name.",
+          "readOnly": true
+        },
+        "controlId": {
+          "type": "string",
+          "description": "The control Id.",
```

Comment:
>ion": "The control Id.", [](http://example.com/codeflow?start=19&length=24)

If your docs say what format is it in, it helps people understand how to actually use the api

Reply:
Fixed, thanks Tim.